src/SystemCommands-RefactoringSupport/SycRefactoringPreview.class.st

```diff
@@ -256,10 +256,13 @@ SycRefactoringPreview >> updateChanges [
 { #category : #update }
 SycRefactoringPreview >> updateTablePresenter [
 	| aCompositeChange |
+	
+	changes ifEmpty: [ self error: 'No changes to apply' ].
+	
 	aCompositeChange := RBCompositeRefactoryChange new.
-	aCompositeChange onSystemDictionary: 
-		(changes at:1 ifAbsent: [ self error: 'No changes to apply' ]) model environment. 
+	aCompositeChange onSystemDictionary: (changes at:1) model environment. 
```

Comment:
the `at:1` could be replaced by `#anyOne`.

It would be nice to add a comment explaining why.
I understand now that the RBCompositeRefactoryChange requires an environment to work. And I assume it is not available? So you get it from one of the changes? (and this assumes all changes are on the same environment?

Did I get it right?

Reply:
Exactly, I was writting a reply for your previous comment but you were faster haha.
Now I removed the error signaling in the last version of the method, because if there are no changes, the CompositeChange won't need an environment anyway

Reply:
> `anyOne`.

Done ;)

> the `at:1` could be replaced by `#anyOne`.

Done ;)